primitives/core/Cargo.toml

```diff
@@ -67,7 +59,7 @@ hex = { version = "0.4", default-features = false, optional = true }
 twox-hash = { version = "1.6.2", default-features = false, optional = true }
 libsecp256k1 = { version = "0.7", default-features = false, features = ["hmac", "static-context"], optional = true }
 merlin = { version = "2.0", default-features = false, optional = true }
-ss58-registry = { version = "1.10.0", default-features = false }
+ss58-registry = { version = "1.11.0", default-features = false }
```

Comment:
Please revert this, I see no reason to bump the version here in this PR.

Reply:
There is no harm.

Reply:
`ss58-registry` need to be updated the latest version (see paritytech/ss58-registry#67) for `parse(try_from_str = Ss58AddressFormat::try_from)`